CONTRIBUTING.md

```diff
@@ -64,7 +64,7 @@ Regular pair programming sessions
 * we team up and work on an issue together (often using Visual Studio live sharing sessions)
 
 Open-Source Community Meeting
-* every Thursday 13:00
+* bi-weekly on Mondays 13:00 (when there is no sprint retrospective)
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-27T10:42:04Z" title="Monday, January 27th 2020, 11:42:04 am +01:00">Jan 27, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -64,7 +64,7 @@ Regular pair programming sessions
 * we team up and work on an issue together (often using Visual Studio live sharing sessions)
 
 Open-Source Community Meeting
-* every Thursday 13:00
+* bi-weekly on Thursday 13:00 (when there is no sprint retrospective)
```


it's `Mondays`
```suggestion
* bi-weekly on Mondays 13:00 (when there is no sprint retrospective)
```